src/test/java/ua/utility/kfsdbupgrade/AppTestBase.java

```diff
@@ -0,0 +1,42 @@
+package ua.utility.kfsdbupgrade;
+
+import java.io.File;
+
+/**
+ * Created by S.G. Skinner on 2017-05-03
+ */
+public abstract class AppTestBase {
+    protected static final String DEFAULT_PROP_FILE =
+            System.getProperty("user.dir")
```

Comment:
Why re-create a DEFAULT_PROPERTIES location that is the same as the main App.java one since we already have this property in the tested App.java?